WeakAurasTemplates/TriggerTemplates.lua

```diff
@@ -239,6 +247,48 @@ local function createTotemTrigger(triggers, position, item)
   end
 end
 
+local function createPowerTrigger(triggers, position, item)
+  triggers[position] = {
+    trigger = {
+      type = "status",
+      event = "Power",
+      unevent = "auto",
+      use_unit = true,
+      unit = "player",
+      use_powertype = true,
+      use_showCost = true,
+      powertype = item.powertype
+    },
+  };
+end
+
+local function createHealthTrigger(triggers, position, item)
+  triggers[position] = {
+    trigger = {
+      type = "status",
+      event = "Health",
+      unit = "player",
+      use_unit = true,
+      unevent = "auto",
+      use_absorbMode = true,
+      use_showAbsorb = true,
+      use_showIncomingHeal = true,
```

Comment:
and here, you do.

Reply:
indeed when you see it, you can't miss them anymore, i added a bunch